app/brave_settings_strings.grdp

```diff
@@ -1411,6 +1411,12 @@ Leo. This action can't be undone.
   <message name="IDS_SETTINGS_LEO_ASSISTANT_SAVE_MODEL_BUTTON_LABEL" desc="A button label for user to click when they want to save a new model">
     Save model
   </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_MODEL_SYSTEM_PROMPT_TITLE" desc="A header title for the model system prompt">
+    System Prompt
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_MODEL_SYSTEM_PROMPT_DESC" desc="A description message for the custom model system prompt">
+    Custom instructions and/or context given to the model to guide its responses in a specific direction or style. Use &#x25;datetime&#x25; where you would like the current date and time to be inserted.
```

Comment:
should we mention that this is optional and that Brave provides a default prompt if it isn't specified?

Reply:
Good call, @fallaciousreasoning! I wonder if we should pre-populate the field with our own default prompt. Then the user can choose to keep it, or modify. Might be even more user-friendly than simply presenting them with an empty textarea...

Reply:
@fallaciousreasoning Thoughts on this approach? I've moved away from using the LLAMA2 prompt, and instead created a dedicated prompt-string for custom models. The first four lines of the prompt are visible in the prompt textarea, demonstrating a default prompt to the user, as well as use of the %datetime% placeholder.

![image](https://github.com/user-attachments/assets/85669c71-cf7f-45c8-a819-d87a22b6badf)